src/components/Pagination/Pagination.scss

```diff
@@ -1,4 +1,7 @@
+$pagination-disabled-bg: transparent;
```

Comment:
Is this another addition due to the var-reset?

Reply:
Actually never mind, I just had to scroll down another 2 inches on the screen

Reply:
No problem; added another commented override in this file to fix the style issue you pointed out in meatspace. I can just TASTE the merge